Sources/NostrSDK/Events/BookmarksListEvent.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  BookmarksListEvent.swift
+//
+//
+//  Created by Bryan Montz on 12/22/23.
+//
+
+import Foundation
+
+/// An event that contains an uncategorized, "global" list of things a user wants to save.
+///
+/// See [NIP-51](https://github.com/nostr-protocol/nips/blob/master/51.md#standard-lists).
+public final class BookmarksListEvent: NostrEvent, HashtagInterpreting, PrivateTagInterpreting {
+    public required init(from decoder: Decoder) throws {
+        try super.init(from: decoder)
+    }
+    
+    @available(*, unavailable, message: "This initializer is unavailable for this class.")
+    override init(kind: EventKind, content: String, tags: [Tag] = [], createdAt: Int64 = Int64(Date.now.timeIntervalSince1970), signedBy keypair: Keypair) throws {
+        try super.init(kind: kind, content: content, tags: tags, createdAt: createdAt, signedBy: keypair)
+    }
+    
+    init(content: String, tags: [Tag] = [], createdAt: Int64 = Int64(Date.now.timeIntervalSince1970), signedBy keypair: Keypair) throws {
+        try super.init(kind: .bookmarksList, content: content, tags: tags, createdAt: createdAt, signedBy: keypair)
+    }
+    
+    /// Ids of bookmarked kind-1 notes.
+    public var noteIds: [String] {
+        allValues(forTagName: .event) ?? []
+    }
+    
+    /// Coordinates of bookmarked articles.
+    public var articlesCoordinates: [EventCoordinates] {
```

Comment:
Would having this class implement the `EventCoordinatesInterpreting` protocol be sufficient to replace this attribute?

Reply:
Yeah, sorry I keep forgetting those. I think I have a compromise that reuses the code and enables custom documentation.